apps/namadillo/src/slices/validators.ts

```diff
@@ -216,22 +237,37 @@ const toMyValidators = (
 const toUnbondingValidators = (
   indexerBonds: IndexerUnbond[],
   totalVotingPower: IndexerVotingPower,
-  unbondingPeriod: bigint
-): MyValidator[] => {
+  unbondingPeriod: bigint,
+  apr: BigNumber
+): MyUnbondingValidator[] => {
+  const timeNow = Math.round(Date.now() / 1000);
+
   return indexerBonds.map((indexerUnbond) => {
     const validator = toValidator(
       indexerUnbond.validator,
       totalVotingPower,
-      unbondingPeriod
+      unbondingPeriod,
+      apr
     );
+    const secondsLeft = BigInt(Number(indexerUnbond.withdrawTime) - timeNow);
+
+    // TODO: later return from the backend
+    const canWithdraw = secondsLeft <= BigInt(0);
+    const timeLeft = timeFromSeconds(canWithdraw ? BigInt(0) : secondsLeft);
```

Comment:
Maybe we should think about adding a date library like `date-fns` to make things easier. Right now I'm getting the following output `0 Days: 9 Hrs: 18 Mins` , I think it should read something like `9h 18 min`